runtime/auth/aws-signing-default/common/test/aws/smithy/kotlin/runtime/auth/awssigning/DefaultMiddlewareSigningTest.kt

```diff
@@ -1,11 +0,0 @@
-/*
- * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0
- */
-package aws.smithy.kotlin.runtime.auth.awssigning
-
-import aws.smithy.kotlin.runtime.auth.awssigning.tests.MiddlewareSigningTestBase
-
-class DefaultMiddlewareSigningTest : MiddlewareSigningTestBase() {
-    override val signer: AwsSigner = DefaultAwsSigner
-}
```

Comment:
**Question**: No more signing middleware testing at all?

Reply:
Relocated to `http-auth-aws`